app/views/provider_interface/hesa_export/new.html.erb

```diff
@@ -1,20 +1,20 @@
-<%= content_for :title, t('page_titles.provider.export_hesa_data', timeframe: '') %>
+<%= content_for :title, t('page_titles.provider.export_hesa_data') %>
 
 <%= breadcrumbs({
   t('page_titles.provider.reports') => provider_interface_reports_path,
-  t('page_titles.provider.export_hesa_data', timeframe: '') => nil,
+  t('page_titles.provider.export_hesa_data') => nil,
 }) %>
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">
-      <%= t('page_titles.provider.export_hesa_data', timeframe: "since #{CycleTimetable.apply_opens.to_s(:govuk_date)}") %>
+      <%= t('page_titles.provider.export_hesa_data') %>
     </h1>
     <p class="govuk-body">
-      The data will include all candidates who have accepted an offer from any of your organisations.
+      The data will include all candidates who have accepted an offer since <%= CycleTimetable.apply_opens.to_s(:govuk_date) %>.
```

Comment:
What if this is after find opens but before apply opens — the date will be in the future?

Reply:
Yes, it will. After asking about this I've been told that this is the reason that we have the `export_hesa_data` feature flag. This feature is made unavailable after find opens until the apply opening date. (The feature flag controls access to this page and generating the report).

https://ukgovernmentdfe.slack.com/archives/CPH8J9G65/p1607685451229900

Addressing this right now does not seem to be high priority so Adam will discuss with James and see if it's worth notifying the providers about the HESA export being unavailable for that period of time, so they can do it prior to the find opening day.